CHANGELOG.rst

```diff
@@ -4,6 +4,13 @@ Changelog
 In development
 --------------
 
+Changed
+~~~~~~~
+
+* Replace "noop" with "continue" when "do" is not specified. The new "continue" command
+  will not alter the previous task state and will continue to conduct the workflow
+  execution. (improvement)
```

Comment:
Should put issue or PR number as reference?

Reply:
Additionally, this may be a breaking change for some workflows, particularly some of ours in st2ci and st2cd.

Reply:
@jinpingh The st2 issue is added to the change log entry.

Reply:
@blag It may in cases where we catch failure then publish but don't do any cleanup/notify/remediation.